apps/condo/domains/common/components/FocusContainer.ts

```diff
@@ -6,4 +6,9 @@ export const FocusContainer = styled.div`
   border: 1px solid ${({ color }) => color || colors.lightGrey[5]};
   border-radius: 8px;
   padding: 24px;
-`
+  
```

Comment:
This changes affect's all other pages. Does design team is agreed this changes?

Reply:
This styles are not from me.